boltdesignsystem/bolt
# Feature/ds 182 blockquote renderer

## Jira

[DS-182](https://pegadigitalit.atlassian.net/browse/DS-182)

## Summary

Convert Bolt-Blockquote to the latest renderer. 

## Details

In an attempt to unify and improve components, I am updating the Blockquote component to use the latest web component renderer. 

## How to test

- Open the blockquote portion of pattern-lab and compare with the origional.
- Review code changes to confirm the conversion is a success.
